### ISSUE TITLE
Shareability of links to Playground and their parameters

### ISSUE THREAD

**Issue by JayPanoz** [edrlab/thorium-web#27]
Right now Playground is only checking for a manifest but longer term, it should support more search parameters e.g. text fragment to share more precisely into an OpenAccess publication for instance. 

We will try listing and documenting all these params in this issue.

**Comment by HadrienGardeur** [comment]
I think that in the case of mobile, shareability could also extend to support the `share()` method as an action: https://developer.mozilla.org/en-US/docs/Web/API/Navigator/share

On desktop, this is not supported in a useful way and should be avoided IMO, in favor of copying to the clipboard.

**Comment by JayPanoz** [comment]
> support the share() method as an action

I'm assuming this could also take different forms? E.g. action, entry in popover menu when highlighting text, buttons in a list of saved highlights, etc.?

So same logic, but the actual output is contextual (publication as a whole, text fragment in a resource, etc.).

**Comment by HadrienGardeur** [comment]
That's right. Initially, we could implement this as an action where the user can include the current position as an option.

The nature of what is shared (URL, text, media…) has an impact on what the native share intent/screen displays to the user on Android and iOS.
